packages/web-components/src/components/cbp-structured-list/cbp-structured-list.scss

```diff
@@ -112,7 +108,7 @@ cbp-structured-list {
     *Collection 
   */
 
-  div[role=list]:only-child > *:first-child {
+  div[role=list] > *:first-child {
```

Comment:
This is now adding the border regardless of the header's existence. It's the same color as the header, but it's adding 2px of border space and affecting the visual alignment (Jeremy will notice).

Reply:
updates as per call